src/index.js

```diff
@@ -0,0 +1,173 @@
+import Alltask from './modules/module1.js';
+import Status from './modules/module2.js';
+
+class TaskList {
+  constructor(doings = []) {
+    this.doings = doings;
+  }
+
+  listFormat() {
+    const todos = document.getElementsByClassName('to-dos');
+
+    if (todos) {
+      todos[0].innerHTML = '';
+    }
+
+    this.doings.forEach((lis) => {
+      const eachList = [];
+      const eachcheck = [];
+      const eachParag = [];
+      const eachbtn = [];
+      const checkNparagDiv = [];
+
+      // create editable List
+      eachList[lis.index] = document.createElement('li');
+      eachList[lis.index].contentEditable = true;
+      eachList[lis.index].setAttribute('id', lis.index);
+      eachList[lis.index].classList.add('indList');
+
+      // create check
+      eachcheck[lis.index] = document.createElement('input');
+      eachcheck[lis.index].setAttribute('type', 'checkbox');
+      eachcheck[lis.index].classList.add('checkbox');
+      eachcheck[lis.index].setAttribute('id', lis.index);
+
+      // create paragraph
+      eachParag[lis.index] = document.createElement('p');
+      /// ///////////////////////////////
+      eachParag[lis.index].textContent = lis.description;
+      eachParag[lis.index].setAttribute('id', lis.index);
+      eachParag[lis.index].classList.add('eachParag');
+
+      // create button
+      eachbtn[lis.index] = document.createElement('button');
+      eachbtn[lis.index].setAttribute('id', lis.index);
+      eachbtn[lis.index].innerHTML = '<i class="fa-solid fa-trash"></i>';
+
+      // Div
+      checkNparagDiv[lis.index] = document.createElement('div');
+      checkNparagDiv[lis.index].append(eachcheck[lis.index], eachParag[lis.index]);
+      eachParag[lis.index].classList.add('checkNparagDiv');
+
+      // appending check, paragraph and button
+      eachList[lis.index].append(eachcheck[lis.index], eachParag[lis.index], eachbtn[lis.index]);
+
+      // appending each list
+      todos[0].append(eachList[lis.index]);
+
+      // On editing paragraph
+      eachParag[lis.index].addEventListener('click', (e) => {
+        e.target.nextSibling.innerHTML = '<i class="fa-solid fa-trash"></i>';
+        e.target.nextSibling.style.cursor = 'pointer';
+        eachList[lis.index].style.backgroundColor = '#a6bdbe';
+        e.target.nextSibling.addEventListener('click', () => {
+          this.delete(eachList[lis.index]);
+        });
+      });
+
+      eachbtn[lis.index].addEventListener('click', () => {
+        this.delete(eachList[lis.index]);
+      });
+
+      eachList[lis.index].addEventListener('mouseleave', (eve) => {
+        eachbtn[lis.index].innerHTML = '<i class="fa-solid fa-trash"></i>';
+        this.update(eve.target.id, eve.target.innerText);
+      });
+
+      // Listen for checking elements
+      eachcheck[lis.index].addEventListener('change', (e) => {
```

Comment:
- [ ] Great job adding all the interactions. Everything works fine. Kindly ensure that when a checkbox is checked, the completed status turns to true. Some change and some don't. Kindly look into it. 
![Screenshot (295)_LI](https://user-images.githubusercontent.com/65298048/184479778-e5d84a5c-01ed-4b62-b045-70e7160d443a.jpg)

Reply:
![Screenshot (421)](https://user-images.githubusercontent.com/96092850/184484261-9e01535c-d54d-4dd2-aa9d-9b19778b92be.png)
DONE

Reply:
DONE